docs/local_installation.md

```diff
@@ -0,0 +1,61 @@
+# Local installation
+
+## Install Requestor
+
+Install required software:
+
+*   [PostgreSQL](PostgreSQL) 9.6 or above
+*   [Python](https://www.python.org/downloads/) 3.7.x
+*   [Poetry](https://poetry.eustace.io/docs/#installation)
+
+Then use `poetry install` to install the dependencies. Before that,
+a [virtualenv](https://virtualenv.pypa.io/) is recommended.
+If you don't manage your own, Poetry will create one for you
```

Comment:
Can we also add a point to `upgrade pip` to latest version before doing a `poetry install`? I personally had problems with installing a few packages until I upgraded my `pip`. wdyt?